Dockerfile

```diff
@@ -10,29 +12,27 @@ ARG FAIL2BAN_GPG_PUBLIC_KEY_ID=0x683BF1BEBD0A882C
 ARG FAIL2BAN_GPG_PUBLIC_KEY_SERVER=keys.gnupg.net
 ARG FAIL2BAN_GPG_FINGERPRINT="8738 559E 26F6 71DF 9E2C  6D9E 683B F1BE BD0A 882C"
 
-LABEL org.opencontainers.image.version=${VCS_VER}
-LABEL org.opencontainers.image.revision=${VCS_REF}
-LABEL org.opencontainers.image.title="docker-mailserver"
-LABEL org.opencontainers.image.vendor="The Docker Mailserver Organization"
-LABEL org.opencontainers.image.authors="The Docker Mailserver Organization on GitHub"
-LABEL org.opencontainers.image.licenses="MIT"
-LABEL org.opencontainers.image.description="A fullstack but simple mail server (SMTP, IMAP, LDAP, Antispam, Antivirus, etc.). Only configuration files, no SQL database."
-LABEL org.opencontainers.image.url="https://github.com/docker-mailserver"
-LABEL org.opencontainers.image.documentation="https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md"
-LABEL org.opencontainers.image.source="https://github.com/docker-mailserver/docker-mailserver"
-
-ENV ENABLE_POSTGREY=0
-ENV FETCHMAIL_POLL=300
-ENV ONE_DIR=0
-ENV POSTGREY_AUTO_WHITELIST_CLIENTS=5
-ENV POSTGREY_DELAY=300
-ENV POSTGREY_MAX_AGE=35
-ENV POSTGREY_TEXT="Delayed by Postgrey"
-ENV SASLAUTHD_MECHANISMS=pam
-ENV SASLAUTHD_MECH_OPTIONS=""
-ENV VIRUSMAILS_DELETE_DELAY=7
-
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Not needed anymore?

Reply:
The `ENV`s can be ommited, the shell remains necessary and is now located at the very top of the file. We need `SHELL` as we're using some Bash features during the build, e.g. `[[ ... ]]`.

Reply:
Ah i missed that `SHELL` at the very top